src/components/EventModal.vue

```diff
@@ -214,25 +258,23 @@ export default {
 					event.addProperty(new TextProperty('DESCRIPTION', this.description))
 				}
 
-				if (this.attendeesList && this.attendeesList.trim() !== '') {
-					const organizerEmail = this.organizerEmail?.toLowerCase() || ''
-
-					const attendeesArray = this.attendeesList
-						.split(',')
-						.map(s => s.trim())
-						.filter((email) => !!email)
+				const organizerEmail = this.organizerEmail?.toLowerCase() || ''
 
-					const filteredAttendees = attendeesArray.filter(att => att.toLowerCase() !== organizerEmail)
+				if (this.attendeesList.length > 0) {
+					if (!organizerEmail) {
+						showError(t('mail', 'You can only invite attendees if your account has an email address set'))
```

Comment:
Feedback comes late *after* the user put in attendees. Disable the attendee selection and show a warning banner earlier.

Reply:
yes

no